backend/src/credentials/listOfRequests.ts

```diff
@@ -1,27 +1,32 @@
 import * as Kilt from '@kiltprotocol/sdk-js'
 
+import { CTYPE_HASH, REQUIRED_PROPERTIES, TRUSTED_ATTESTERS } from '../config'
+
+// Here you can set which type of credential (cType) your dApp will request users to login.
+// You can change it by importing a different one.
+// The default is the Email CType by SocialKYC and SocialKYC as the Issuer
 // Establish which cTypes our dApp accepts and which attesters we trust:
 
+const trustedAttestersValues = TRUSTED_ATTESTERS.split(',')
+const requiredPropertiesValues = REQUIRED_PROPERTIES.split(',')
+
+const requiredProperties = requiredPropertiesValues.map(
```

Comment:
What is this snippet of code doing exactly?

Reply:
This comment still not addressed.

Reply:
it was a experimental change to accept more than one TRUSTED_ATTESTERS which separated by a comma